tests/cacheKey.test.js

```diff
@@ -0,0 +1,9 @@
+import { expect, it } from '@jest/globals'
+import { hashQueryKey } from '../index'
+
+it('should hash a array with objects', () => {
+  expect(hashQueryKey(['test', ['hallo'], { a: 1, b: 'b' }])).toEqual(hashQueryKey(['test', ['hallo'], { b: 'b', a: 1 }]))
```

Comment:
Je wilt niet de losse functie testen. Je wilt het effect van je hash functie op je cache testen.

Daarbij is het belangrijk dat je eerst een falende test hebt voordat je in je cache de `hashQueryKey` aanroept.